setup.py

```diff
@@ -33,19 +33,25 @@
 import vsc.install.shared_setup as shared_setup
 from vsc.install.shared_setup import ag, sdw
 
+install_requires = [
```

Comment:
is this not cleaner in a function?

Reply:
Why does that matter at all?

You'll have to call the function anyway to populate the `setup` call?

It's not like you can import the `setup.py` rather than run it, that doesn't make sense.